ogc/enums.py

```diff
@@ -32,16 +32,18 @@
         },
     },
     "google": {
-        "arm64": {},
-        "amd64": {
-            "ubuntu-latest": "ubuntu-2004-focal-v20220325",
-            "ubuntu-2004": "ubuntu-2004-focal-v20220325",
-            "ubuntu-1804": "ubuntu-1804-bionic-v20220325",
-            "sles-latest": "sles-15-sp3-v20220223",
-            "sles-15": "sles-15-sp3-v20220223",
-            "debian-latest": "debian-11-bullseye-v20220317",
-            "debian-10": "debian-10-buster-v20220317",
-            "debian-9": "debian-9-stretch-v20220317",
-        },
+        "ubuntu-latest-lts-arm64": "ubuntu-2204-jammy-arm64-v20230429",
+        "ubuntu-2204-lts-arm64": "ubuntu-2204-jammy-arm64-v20230429",
+        "ubuntu-2004-lts-arm64": "ubuntu-2004-focal-arm64-v20230302",
+        "ubuntu-1804-lts-arm64": "ubuntu-1804-bionic-arm64-v20230510",
+        "ubuntu-latest-lts": "ubuntu-2204-jammy-v20230429",
+        "ubuntu-2204-lts": "ubuntu-2204-jammy-v20230429",
+        "ubuntu-2004-lts": "ubuntu-2004-focal-v20230302",
+        "ubuntu-1804-lts": "ubuntu-1804-bionic-v20230510",
+        "sles-latest": "sles-15-sp3-v20220223",
+        "sles-15": "sles-15-sp3-v20220223",
+        "debian-latest": "debian-11-bullseye-v20220317",
+        "debian-10": "debian-10-buster-v20220317",
+        "debian-9": "debian-9-stretch-v20220317",
```

Comment:
I am not a fan of this, I tried to get the `ex_get_image_from_family` to work, but it doesn't seem to work. So the images had to be hard coded. This should really be fixed to just have `ubuntu-2204-lts` be the latest image, instead of hard coded here. Anyway it works for now with this.